_docs/setup/kubernetes/mesh-expansion.md

```diff
@@ -100,11 +102,13 @@ and install the setup:
 # Check what the script does to see that it meets your needs.
 # On a Mac, either brew install base64 or set BASE64_DECODE="/usr/bin/base64 -D"
 export GCP_OPTS="--zone MY_ZONE --project MY_PROJECT"
+```
+```bash
 install/tools/setupMeshEx.sh machineSetup VM_NAME
 ```
 
 Or the equivalent manual steps:
-
+------- Manual setup steps begin -------
```

Comment:
how does this look like with jekyll ? screen shot ?

Reply:
http://0.0.0.0:4000/docs/setup/kubernetes/mesh-expansion.html

Just in case my server dies, sorry for non-google reviewers, who would not be able to access this page:
https://screenshot.googleplex.com/euT65Eq4PsW
https://screenshot.googleplex.com/hkZgqZ3a6GL

Reply:
0.0.0.0 won't work for anyone :)

Pro tip: you can drop screenshot directly into github